internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -179,25 +179,30 @@ Resources:
     Metadata:
       aws:copilot:description: A custom resource to trigger the file-copying state machine
     Type: Custom::TriggerStateMachine
-    DependsOn: 
-      - CopyAssetsStateMachine {{- /* This is a real dependency */}}
+    DependsOn:
       - Bucket
       - BucketPolicyForCloudFront
       - CloudFrontOriginAccessControl
       - CloudFrontDistribution
       - TriggerStateMachineFunction
       - TriggerStateMachineFunctionRole
-      - CopyAssetsStateMachine
+      - CopyAssetsStateMachine {{- /* This is a real dependency */}}
       - CopyAssetsStateMachineRole
```

Comment:
Do we still need this?

Reply:
yeah, still only want to trigger the state machine _after_ everything else finishes

Reply:
no i meant do we need this resource?

Reply:
i did remove these two, if that's what you're thinking https://github.com/aws/copilot-cli/pull/4888/files#diff-15ae3d1d11e7d786765fd0df1a1cdd256c6083500fc8c1e0f3856ffa7c4716f9L193-L194. They were similarly named :)

Reply:
oh that's what i thought 🤦